lib/element/locator.js

```diff
@@ -301,7 +301,12 @@ class LocateElement {
 
     const errorResult = this.transport.getElementNotFoundResult(result);
 
-    throw new Error(`Element ${element.toString()} not found.${errorResult.message ? (' ' + errorResult.message) : ''}`);
+    return {
+      status: errorResult.status,
+      value: null,
+      error: errorResult.errorStatus,
+      message: errorResult.message
+    };
```

Comment:
Doing this would also change the flow for the `assert.visible()` and `expect.element().visible` commands where getting an error is the expected flow.

The `expect.element().not.present` here should take a similar flow as `assert.not.elementPresent()` which happens to work fine.